core/storage/auth/gae_models.py

```diff
@@ -254,16 +285,20 @@ def get_export_policy(cls):
 
     @classmethod
     def apply_deletion_policy(cls, user_id):
+        # type: (Text) -> None
         """Delete instances of UserIdByFirebaseAuthIdModel for the user.
 
         Args:
             user_id: str. The ID of the user whose data should be deleted.
         """
-        datastore_services.delete_multi(
-            cls.query(cls.user_id == user_id).fetch(keys_only=True))
+        firebase_models = cls.query(
```

Comment:
This variable name should probably be firebase_model_keys (and, do the cast inline).

Reply:
I was thinking for a common naming convention throughout and named it `keys = .. `in all the cases and in the next PR too.